source/rust_verify/example/multiset.rs

```diff
@@ -0,0 +1,51 @@
+// examples of using the multiset, sorted_by lemmas in seq lib
+use vstd::prelude::*;
+use vstd::seq_lib::*;
+use vstd::relations::sorted_by;
+
+verus! {
+
+proof fn multiset_ext_eq() {
+
+    let a: Seq<int> = seq![1, 2, 3];
+    let b: Seq<int> = seq![1, 3, 2];
+
+    lemma_seq_properties::<int>(); //Provides the necessary lemmas for seq to multiset conversion
+    assert(a.to_multiset() =~= b.to_multiset());
+}
+
+proof fn multiset_ext_eq2() {
+
+    let a: Seq<int> = seq![3, 2, 1, 1, 2, 3];
+    let b: Seq<int> = seq![1, 2, 3, 1, 2, 3];
+    
+    lemma_seq_properties::<int>(); //Provides the necessary lemmas for seq to multiset conversion
+    assert(a.to_multiset() =~= b.to_multiset());
+}
+
+proof fn sorted_by_eg() {
```

Comment:
"eg" -> "leq"?

Reply:
I think the point was it's an example of using the function sorted_by. But I can change it if you think that is more appropriate

Reply:
Yes, I think the `eg` looks odd without the periods and in a non-formal writing setting.